misc/docker-compose.yml

```diff
@@ -33,7 +33,9 @@ services:
     volumes:
       - uta_vol:/var/lib/postgresql/data
     network_mode: host
-
+    environment:
+      - POSTGRES_HOST_AUTH_METHOD=trust
```

Comment:
Why is this needed? You should be able to log into all instances with anonymous:anonymous.

Reply:
removed it.